### PR TITLE
feat(debug): add congestion control debug page

### PR DESCRIPTION
**Pull request by Trisfald**
Adding a new debug page for congestion control at `/debug/pages/congestion_control`. 
It shows the list of most recent blocks and chunks. There is information about various parameters that can tell the level of congestion across shards. 

No data is shown if the congestion control feature is not enabled.

### CODE REVIEW

**Comment by Trisfald** [comment]
Preview (no congestion):
congestion_control

**Comment by wacban** [comment]
@jakmeier We have debug page now, we are officially ready to launch ;)

**Review by wacban — APPROVED** [review]
LGTM

File: chain/jsonrpc/res/congestion_control.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
+function sortBlocks(blocks) {
+    function sortingKey(row) {
+        return row.block_height + (row.block_timestamp / 1e12 % 1);
```
[the remark this answers is not in the document]
Comment (Trisfald) [reply chain/jsonrpc/res/congestion_control.js]: I failed to find a reason for `% 1` to be there, so I removed it

File: chain/client-primitives/src/debug.rs [hunk 2]
```diff
@@ -42,6 +42,14 @@ pub struct DebugChunkStatus {
...
     #[serde(skip_serializing_if = "Option::is_none")]
     pub processing_time_ms: Option<u64>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub delayed_receipts_gas: Option<u128>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub buffered_receipts_gas: Option<u128>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub receipt_bytes: Option<u64>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub allowed_shard: Option<u16>,
```
[the remark this answers is not in the document]
Comment (Trisfald) [reply chain/client-primitives/src/debug.rs]: good point, done!

**Comment by jakmeier** [comment]
Great to have this!

Should we also add the congestion level (value between 0 and 1) per shard on this page? (Can be in another PR)

File: chain/jsonrpc/res/congestion_control.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jakmeier) [reply chain/jsonrpc/res/congestion_control.js]: I think the `% 1` is there for a reason.

Looks like this code wants to sort rows by block height and block timestamp. Using `% 1` keeps only the fractional part, to ensure the timestamp is strictly weaker than the block height in comparison.

For example, if we have a timestamp that like 111_222_333_444_555_666_777_888_999 (in ns), after `/ 1e12` we have `111_222_333.444_555_666_777_888_999` (dot between 3s and 4s). The `% 1` turns it into `0.444_555_666_777_888_999`.

Not sure if can really matter in practice. But maybe we should keep it and add a short comment?

**Review by wacban — APPROVED** [review]
LGTM!

**Pull request merged**